packages/amazonq/test/e2e_new/amazonq/utils/generalUtils.ts

```diff
@@ -117,7 +117,7 @@ export async function waitForChatResponse(webview: WebviewView, timeout = 15000)
  * @param webview The WebviewView instance
  * @returns Promise<boolean> True if successful, false if an error occurred
  */
-export async function clearChat(webview: WebviewView): Promise<boolean> {
+export async function clearChatInput(webview: WebviewView): Promise<boolean> {
```

Comment:
nice, I like how this disambiguates between it clearing the entire chat window and the chat contents itself.